docs/collecting-data/collecting-from-own-applications/java-tracker/previous-versions/java-tracker-v0-12/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Java Tracker v0.12"
+date: "2020-02-26"
+sidebar_position: 100
+---
+
+```mdx-code-block
+import Badges from '@site/src/components/Badges';
+
+<Badges badgeType="Early Release"></Badges>
+```
+
+The Snowplow Java Tracker lets you add analytics to your [Java](http://www.java.com/en/)\-based desktop and server apps, servlets and games.
+
+The Tracker should be relatively straightforward to setup if you are familiar with Java development.
+
+:::info These pages are for version 0.12
```

Comment:
I suggest to run `./make-links-validatable.sh` to make sure `index.md` is not missing in this and other links

Reply:
Done!